### ISSUE TITLE
Mail Delivery logs for Virtulamin

### ISSUE THREAD

**Issue by shoulders** [virtualmin/virtualmin-gpl#945]
| SYSTEM INFORMATION ||
|----------|----------|
|  OS type and version |  Ubuntu Linux 22.04.5              |
|  Usermin version     |  2.102                             |
|  Virtualmin version  |  7.20.2 Pro                        |
|  Theme version       |  21.20.7                           |
|  Apache version      |  2.4.52                            |
|  Package updates     |  46 package updates are available  |

## the request

I would like a mail deilvery / track delivery option for emails. This is an incredible useful for a user check if mail has been delivered.

this could be a pro feature

![image](https://github.com/user-attachments/assets/a74c2b8e-413d-4ec2-ac19-418b5d87c735)

**cPanel**

![image](https://github.com/user-attachments/assets/60a62625-e3c8-4779-b55e-89cbe98e670a)

![image](https://github.com/user-attachments/assets/2417d949-d9ce-4ac2-a131-01150007a699)

![image](https://github.com/user-attachments/assets/5c78f8d5-da05-418a-902c-15c05ff7a040)

**Comment by jcameron** [comment]
Would the implementation of https://github.com/virtualmin/virtualmin-gpl/issues/948 provide this feature for you?

**Comment by shoulders** [comment]
I dont think so as the cpanel has more information, however this would be the place for it.

Maybe an upgrade?

I will see what the results look like tommorow.

**Comment by iliaross** [comment]
Reopened, as this was closed by an automated cleanup run. @shoulders, sorry for the noise, and thanks for the contribution. We definitely didn't mean any disrespect.

If it turns out the issue is already resolved for you, you're welcome to close it.
